### PR TITLE
Make different settings values per track possible.

### PR DESCRIPTION
**Pull request by jan-ivar**
Fix for https://github.com/w3c/mediacapture-main/issues/466.

### CODE REVIEW

File: getusermedia.html [hunk 1]
```diff
@@ -99,17 +99,17 @@ <h2>Terminology</h2>
...
-        two different track objects bound to the same source ask for the same
-        capability or setting information, they will get back the same
-        answer).</p>
+        media that conforms to the constraints present on that track, to that
+        track. Multiple tracks can be attached to the same source. User Agent
+        processing, such as downsampling, MAY be used to ensure that all tracks
+        have appropriate media.</p>
+        <p>Sources have <code><a>capabilities</a></code>, which are manipulated
+        through constrainable properties on the track, and observed by
+        <code><a>settings</a></code> on the track. The constrainable properties
```
Comment (burnburn) [inline getusermedia.html]: Not sure I agree with changing the semantics here to say that sources have capabilities, and that those capabilities are manipulated via constrainable properties.  I still think the fundamental unit here is the property, not the capability.

**Comment by alvestrand** [comment]
Needs more review.

**Comment by jan-ivar** [comment]
OK this should hopefully be better. @burnburn PTAL.

**Comment by burnburn** [comment]
@alvestrand editors agree you can merge if you are happy with this.

**Pull request merged**